.gitignore

```diff
@@ -76,6 +76,7 @@ sorc/gfs_post.fd
 sorc/gfs_wafs.fd
 sorc/gldas.fd
 sorc/gsi.fd
+sorc/gdas.cd
```

Comment:
typo?

Reply:
Per NCO implementation standards `.fd` indicates a directory with mainly fortran source, `.cd` with mainly C source. I made the executive decision that JEDI is 'more C than F'. :-)